### PR TITLE
feat: remove wallet bridge

### PR DESCRIPTION
**Pull request by samsiegart**
fixes https://github.com/Agoric/dapp-inter/issues/201

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -147,14 +135,13 @@ const App = () => {
...
       ></ToastContainer>
-      <OfferSignerBridge />
       <div className="w-screen max-w-7xl m-auto">
         {error ? (
           <>
             <div className="flex justify-end flex-row space-x-2 items-center mr-6 m-2">
               {networkDropdown}
             </div>
-            <div>Error connecting to chain</div>
+            <div>Error connecting to chain!</div>
```
Comment (turadg) [inline src/App.tsx:144]: lol

File: src/components/CloseVaultDialog.tsx [hunk 2]
```diff
@@ -130,7 +120,7 @@ const CloseVaultDialog = ({
           {...noticeProps}
           className="overflow-hidden text-interOrange"
         >
-          Offer sent to Agoric Smart Wallet for approval.
+          Vault has been closed successfully.
```
Comment (turadg) [inline src/components/CloseVaultDialog.tsx:123]: It's only been sent, right? The transaction could still fail. For example if another transaction in flight took out more debt, this pay down to close would not be enough.

File: src/components/ManageVaults.tsx [hunk 3]
```diff
@@ -111,7 +113,7 @@ const ManageVaults = () => {
     );
   } else if (!chainConnection) {
     popupContent = 'Connect your wallet to manage your vaults.';
-  } else if (vaults?.size === 0) {
+  } else if (vaults?.size === 0 || isSmartWalletProvisioned === false) {
```
Comment (turadg) [inline src/components/ManageVaults.tsx:116]: Consider having its own case message. Perhaps "You have no Agoric smart wallet yet."

File: src/components/ProvisionSmartWalletDialog.tsx [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+
+  const provision = () => {
+    assert(chainConnection);
+    provisionSmartWallet(chainConnection, setProvisionToastId);
+    setShouldBeOpenIfFeeLoaded(false);
+  };
+
+  const body = (
+    <span>
+      To interact with contracts on the Agoric chain, a smart wallet must be
```
Comment (turadg) [inline src/components/ProvisionSmartWalletDialog.tsx:98]: For future work: should we have a utility for this in ui-kit? Worth an issue imo

File: src/components/VaultAdjustmentDialog.tsx [hunk 5]
```diff
@@ -9,30 +7,24 @@ type Props = {
   onClose: () => void;
 };
 
-const message =
-  'Your vault adjustment request has been successfully submitted. Go to your Agoric Smart Wallet to approve.';
+const message = "Your vault's balances have been updated.";
```
Comment (turadg) [inline src/components/VaultAdjustmentDialog.tsx:10]: Same race issue as above.

File: src/components/VaultCreationDialog.tsx [hunk 6]
```diff
@@ -9,30 +7,24 @@ type Props = {
...
-
   const goToVaults = () => {
     onClose();
     setViewMode(ViewMode.Manage);
   };
 
   return (
     <ActionsDialog
-      title="Success: Offer Submitted"
+      title="Success: Vault Created"
```
Comment (turadg) [inline src/components/VaultCreationDialog.tsx:22]: This may be okay sinxe the 'message' is accurate

File: src/service/vaults.ts [hunk 7]
```diff
@@ -425,157 +426,148 @@ export const makeOpenVaultOffer = async (
...
+      }
+    },
+  );
 };
 
 type Proposal = {
   give: {
-    Collateral?: { amount: CapData<string> };
-    Minted?: { amount: CapData<string> };
+    Collateral?: Amount;
```
Comment (turadg) [inline src/service/vaults.ts:463]: Consider adding 'nat'

**Review by turadg — COMMENTED** [review]
Huge UX improvement. Want to make sure the messages are accurate before approving.

File: src/components/CloseVaultDialog.tsx (hunk 2 above)
Comment (turadg) [reply src/components/CloseVaultDialog.tsx:123]: Oh just the variable name is off. Approving assuming you'll revise

**Review by turadg — APPROVED** [review]

File: src/service/vaults.ts (hunk 7 above)
Comment (samsiegart) [reply src/service/vaults.ts]: Done

File: src/components/ProvisionSmartWalletDialog.tsx (hunk 4 above)
Comment (samsiegart) [reply src/components/ProvisionSmartWalletDialog.tsx:98]: Possibly for the provisioning fee, though I'm not sure how likely that flow is to change (i.e. if we implement automatic provisioning). We already have a utility for the provision status. I'm not sure if the whole dialog flow is worth making into a component depending on how dapps want to handle it

File: src/components/ManageVaults.tsx (hunk 3 above)
Comment (samsiegart) [reply src/components/ManageVaults.tsx:116]: I considered that but I'm leaning towards keeping this as is. We already show the dialog, so it's not necessary to tell them twice, and it doesn't matter to them that there's no smart wallet until they actually try to create a vault, where they'll see the "Provision" button in that view.

File: src/components/CloseVaultDialog.tsx (hunk 2 above)
Comment (samsiegart) [reply src/components/CloseVaultDialog.tsx:123]: Updated the variable name

**Pull request merged**